recipes/scanpy/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "scanpy" %}
+{% set version = "1.7.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ad5e7afdf1948753cd27f6f27ee904df1ce455855eec8520985effe86c53c630
+
+build:
+  # noarch: python  # for some reason the [py<38] makes it not work?
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - flit-core
+    - setuptools_scm
+    - pytoml
+  run:
+    - python
+    - anndata >=0.7.4
+    - numpy >=1.17
+    - matplotlib >=3.1.2
+    - pandas >=0.21
+    - scipy >=1.4
+    - seaborn
+    - h5py >=2.10
+    - pytables  # “tables” on PyPI
+    - tqdm
+    - scikit-learn >=0.21.2
+    - statsmodels >=0.10
+    - patsy
+    - networkx >=2.3
+    - natsort
+    - joblib
+    - numba >=0.41
+    - umap-learn >=0.3.10
+    - legacy-api-wrap
+    - packaging
+    - sinfo
+    - importlib_metadata  # [py<38]
```

See https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages
```suggestion
    - importlib_metadata
```

---

I see

> Currently available packages:
> 
> Name | Available on: | Empty on:
> -- | -- | --
> dataclasses | python &gt;=3.6,&lt;3.7 | python &gt;=3.7
> typing | &nbsp; | python &gt;=3

no mention of `importlib_metadata`.

---

@chrisburr ping